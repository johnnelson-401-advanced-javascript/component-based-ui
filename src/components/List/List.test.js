import { shallow } from 'enzyme';
import ListItems from './List';

describe('List.js', () => {
  it('Renders a List of Characters', () => {
    const characters = [{
      'name': 'Noob-Noob',
      'status': 'Alive',
      'species': 'Poopybutthole',
      'image': 'https://rickandmortyapi.com/api/character/avatar/252.jpeg'
    },
    {
      'name': 'Noob-Noob',
      'status': 'Alive',
      'species': 'Poopybutthole',
      'image': 'https://rickandmortyapi.com/api/character/avatar/252.jpeg'
    }];
    let wrapper = shallow(ListItems({ characters }));
    expect(wrapper).toMatchSnapshot();
  });
  
});
