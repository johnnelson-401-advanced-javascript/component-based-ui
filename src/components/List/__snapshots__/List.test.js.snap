// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List.js Renders a List of Characters 1`] = `
<ul
  className="listDisplay"
>
  <li
    key="Noob-Noob"
  >
    <Character
      image="https://rickandmortyapi.com/api/character/avatar/252.jpeg"
      name="Noob-Noob"
      species="Poopybutthole"
    />
  </li>
  <li
    key="Noob-Noob"
  >
    <Character
      image="https://rickandmortyapi.com/api/character/avatar/252.jpeg"
      name="Noob-Noob"
      species="Poopybutthole"
    />
  </li>
</ul>
`;
